.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: trade-tariff/process-appendix-5a/ci
+on:
+  push:
+    branches-ignore:
+    - main
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: python:3.11.2
```

Comment:
this image version is newer than the test one. Is there a reason for that?

Reply:
It'll be a very small risk doing the upgrade on such simple python code with a small subset of deps

Reply:
Sure, but why introduce this discrepancy in the first place is my question 👀